### ISSUE TITLE
Coding Style

### ISSUE THREAD

**Issue by hideon-bush** [alecjacobson/computer-graphics-raster-images#64]
Are we being graded on the coding style for the labs?

**Comment by Jack-XHP** [comment]
nope, but in case your code doesn't run successfully, a good coding style will make it easier to understand by TA.
